content/blog/2020/09/2020-09-09-jenkins-continuous-evolution-cdcon.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Learn more about Jenkins' continuous evolution at CDCon"
+tags:
+- cdcon
+- talks
+- jcasc
+- cicd
+- tekton
+author: cdfoundation
+description: "Don't miss the Jenkins talk at CDCon, October 7-8, 2020."
+opengraph:
+  image: /images/post-images/2020/jenkins-cdcon.png
+---
+
+The Jenkins project has been around for over fifteen years and is the defacto platform for CI/CD. One of the reasons it continues to be so ubiquitous is that Jenkins constantly evolves and offers flexibility to integrate other tools that work well for your solution. 
+
+At link:https://events.linuxfoundation.org/cdcon/[CDCon], on October 7-8, there are two particular Jenkins talks that will focus on new directions that the Jenkins platform is evolving too and getting better and better for users.
+
+== link:https://sched.co/dqOc[Heard of JCasC and Not Sure Where to Start? Let me Help You!]
+Configuration as code is a best practice for your CI/CD setup as it makes the complex process of setting up Jenkins simpler and more reproducible. Jenkins Configuration as Code (JCasc) enables Jenkins users to define the whole configuration as a simple, plain text YAML syntax. With JCasc, setting up a new Jenkins master is easier than ever before. To get started with JCasC some initial effort is required. This talk walks you through a basic setup for easily spinning up new Jenkins instances. 
+
+October 7 at 2:20 PM PST
+Speaker: Ewelina Wilkosz, Eficode
+
+Ewelina W is passionate about making sure that her customers' software is being built, tested and released in the best possible way. And, most importantly, that software developers don’t hate the process. Ewelina has been involved in Jenkins Configuration as Code plugin development from the very beginning. This is a must-see talk where Ewelina will also share some tips and tricks. The talk will feature using Docker, Jenkins and GitHub Actions as a quick way to build… Jenkins! 
+
+== link:https://sched.co/dpvN[Bridging the Gap with Tekton-client-plugin for Jenkins]
+Tekton provides Kubernetes-native CI/CD building blocks. It enables users to take full advantage of cloud-native features around scalability and high availability. Jenkins flexibility enables integration with Tekton. This talk showcases the new tekton-client-plugin for Jenkins that enables Jenkins to interact with Tekton pipelines on a Kubernetes cluster. Tekton and Jenkins are both CDF projects and this talk highlights the first steps towards better Tekton and Jenkins interoperability, a key goal of the CD Foundation.
+
+October 7 at 11:40 AM PST
+Speaker: Vibhav Bobade, Red Hat
+
+=== Register for CDCon
```

Comment:
Since the other two headings are links, should this be a link also?

```suggestion
=== link:https://events.linuxfoundation.org/cdcon/[Register for CDCon]
```

Reply:
Fixed in 00cdbcf2744d1334815b419c0f31df7473a1b46a